docs/en/Running-Presto-with-Alluxio.md

```diff
@@ -88,6 +88,11 @@ Alternatively, you can also append the conf path (i.e. `/<PATH_TO_ALLUXIO>/conf`
 Also, it's recommended to increase `alluxio.user.network.netty.timeout` to a bigger value (e.g. `10min`) to avoid the timeout
  failure when reading large files from remote worker.
 
+#### Enable `hive.force-local-scheduling`
+
+It's recommended to collocate Presto with Alluxio so that Presto workers can read data locally. An important option to enable is `hive.force-local-scheduling`, which force splits to be 
```

Comment:
- "It's" -> "It is" 
- "force" -> "forces"
- Telling ppl `hive.force-local-scheduling` is a Presto conf (rather than Alluxio)?